appservice/src/deploy/localGitDeploy.ts

```diff
@@ -34,7 +36,7 @@ export async function localGitDeploy(client: SiteClient, fsPath: string): Promis
         }
         await verifyNoRunFromPackageSetting(client);
         ext.outputChannel.appendLine(formatDeployLog(client, (localize('localGitDeploy', `Deploying Local Git repository to "${client.fullName}"...`))));
-        await tryPushAndWaitForDeploymentToComplete();
+        await callWithMaskHandling(async (): Promise<void> => { await tryPushAndWaitForDeploymentToComplete(); }, publishingPassword);
```

Comment:
I still want this around all of localGitDeploy just to be safe. I don't like picking and choosing the lines that we mask